Docker_run_cuda_10_0

```diff
@@ -0,0 +1,31 @@
+ARG BUILD_IMAGE_NAME
+FROM ${BUILD_IMAGE_NAME} AS builder
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04
+
+ARG PYVER=2.7
+ENV PYVER=${PYVER}
```

Comment:
Definitely we want to have ability to create container for various python versions inside.

Reply:
Mhh, maybe I'm missing something but this way we still may have different images for different python versions - this docker file just makes them easy to create.
I don't understand how it will lead to "multiple pythons in one system". We have separate image for each python and each CUDA version.
That is right, we remove existing python and install one we wan to test here.
So main goal of this dokcer is to provide easy way to test various python builds, even if user will prefer to use 2.7 or 3.6 only.